package.json

```diff
@@ -31,7 +31,7 @@
   "author": "LangChain",
   "license": "MIT",
   "resolutions": {
-    "@langchain/core": "0.1.51"
```

Comment:
why do we need this in both package json files?

Reply:
Not 100% sure, but my assumption is because the `@langchain/core` dependencies are also needed in the `examples` directory, which is at the same level as `langgraph`.